### ISSUE TITLE
aria-readonly: Consider explicitly calling out UIA mapping for controls that don't follow the ValuePattern

### ISSUE THREAD

**Issue by sivakusayan** [w3c/core-aam#127]
As discussed in https://github.com/w3c/aria/issues/1309#issuecomment-1166059034, UIA doesn't have a mapping for readonly checkboxes, even though [`aria-readonly` is allowed on checkboxes](https://w3c.github.io/aria/#aria-readonly). Readonly semantics end up being exposed through the `AriaProperties` property instead. Should the AAM be updated to call this out in the `aria-readonly` table? It seems like that is done for cases like [`aria-sort`](https://w3c.github.io/core-aam/#ariaSortAscending).

**Comment by spectranaut** [comment]
Sorry for only replying to this one as well, a year later! This is also a reasonable and useful change. Let me know if you would like to try making the change, otherwise I'll put it on my to dos.
